build/install-istio-with-kind.sh

```diff
@@ -17,15 +17,16 @@ if [ -z $(kind get clusters) ]; then
     kind create cluster
 fi
 
-# Get kubeconfig
-export KUBECONFIG="$(kind get kubeconfig-path --name=kind)"
-
 # Download and install kubectl
 curl -LO \
 https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/${GOOS}/${GOARCH}/kubectl && chmod +x ./kubectl && sudo mv kubectl /usr/local/bin/
 
+# Use kind cluster
+kubectl cluster-info --context kind-kind
+
 # Download and install Istio
 curl -L https://git.io/getLatestIstio | ISTIO_VERSION=${ISTIO_VERSION} sh - && mv istio-${ISTIO_VERSION} /tmp
-cd /tmp/istio-${ISTIO_VERSION}
+pushd /tmp/istio-${ISTIO_VERSION}
 bin/istioctl install -y --set profile=demo
+popd
```

Comment:
This `cd` to `pushd` change doesn't really matter for the script, but it also doesn't hurt...